### DISCUSSION TITLE
The `logFormat` configuration does not appear to be possible

### DISCUSSION THREAD

**Discussion by jamietanna** [renovatebot/renovate#18967] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

2.6.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

GitHub.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

The `logFormat` configuration option is noted in [the documentation for `logContext`](
https://github.com/renovatebot/renovate/blob/848dc84244bbb41cedabd2329931a6d563fe6f97/docs/usage/self-hosted-configuration.md?plain=1#L443-L446), but does not appear to be found when searching through the codebase, so may no longer be possible?

### Relevant debug logs

N/A

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
You can set `LOG_FORMAT=json` in env. A PR to improve documentation would be appreciated if you'd like

**Reply by jamietanna** [reply]
Ah perfect, thank you 🙌 I thought I'd seen that somewhere ([docs](https://github.com/renovatebot/renovate/blob/848dc84244bbb41cedabd2329931a6d563fe6f97/docs/usage/examples/self-hosting.md?plain=1#L385-L392)).

I'll get a PR raised to document that 👍
